source/elm-live.js

```diff
@@ -112,7 +112,7 @@ ${ indent(String(elmMake.error), '  ') }
   }
 
   // Watch Elm files
-  const watcher = chokidar.watch('**/*.elm', { ignoreInitial: false, ignorePermissionErrors: true });
+  const watcher = chokidar.watch('**/*.elm', { ignoreInitial: true, ignorePermissionErrors: true });
```

Comment:
I set `ignorePermissionErrors` on the assumption that it would be better to have a smoother experience than to be unable to use the tool because there are files that can't be accessed.  But I'll go ahead and turn it off, until someone complains, then we can decide the best path forward.